app/assets/stylesheets/user_landing.scss

```diff
@@ -0,0 +1,64 @@
+@import 'global.scss';
+
+.header-landing {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+
+  &__title {
+    color: $primary_color;
+    font-size: 40px;
+    padding: 1%;
+  }
+
+  &__subtitle {
+    font-size: 15px;
+  }
+}
+
+.prompt-container {
+  margin-top: 5%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+}
+
+.prompt {
+  text-align: center;
+
+  &__btn, input[type="submit"] {
+    margin: 3%;
+    min-width: 187px;
+    padding: 13px 25px;
+    border: 0;
+    border-radius: 100px;
+    font-size: 18px;
+    background-color: $primary_color;
+    color: $secondary_color;
+  }
+
+  &__input,
+  input[type="text"] {
+    outline: none;
+    text-align: center;
+    font-size: 35px;
+    border-top: none;
+    border-left: none;
```

Comment:
Unexpected value "none" for property "border-left"    declaration-property-value-blacklist